src/main/java/org/fcrepo/importexport/ArgParser.java

```diff
@@ -130,6 +130,13 @@
                 .desc("When present this flag indicates that external content should be exported.")
                 .required(false).build());
 
+        // Retrieve inbound references
+        configOptions.addOption(Option.builder("i")
```

Comment:
What is the expected behavior when the `-i` option is provided during an `import`?